lib/oauth2-client.js

```diff
@@ -0,0 +1,185 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+import database from 'bedrock-mongodb';
+import * as bedrock from 'bedrock';
+import {httpClient} from '@digitalbazaar/http-client';
+import logger from './logger.js';
+const {util: {BedrockError}} = bedrock;
+
+bedrock.events.on('bedrock-mongodb.ready', async () => {
+  await database.openCollections(['access-tokens']);
```

Comment:
This collection name doesn't follow our naming convention. It should be: `<bedrock module name prefix>-<camel case local collection name that reflects a singular doc name>` (eg: https://github.com/digitalbazaar/bedrock-tokenization/blob/master/lib/documents.js#L22). So this should be: `oauth2-client-accessToken`.